www/templates/intro/login.html

```diff
@@ -3,7 +3,7 @@
 <center><h3>Login</h3></center></div>
 
 <div class="intro-text">
-Please log-in with the code provided to you by a UC Berkeley Researcher.
+Please log-in with the email provided to you by a UC Berkeley Researcher.
```

Comment:
Shouldn't this still be `code`? At least looking at the consent document, you are not going to use emails.

Reply:
Emails is correct thank you